receiver/fluentforwardreceiver/factory.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fluentforwardreceiver
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configerror"
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/consumer"
+)
+
+// This file implements factory for SignalFx receiver.
+
+const (
+	// The value of "type" key in configuration.
+	typeStr = "fluentforward"
+)
+
+// Factory is the factory for SignalFx receiver.
+type Factory struct {
+}
+
+var _ component.ReceiverFactory = (*Factory)(nil)
```

Comment:
I may be wrong but I think it is not necessary to implement this interface if you only need to support logs.

Reply:
You're right -- I removed the extraneous methods.